Dockerfile

```diff
@@ -3,18 +3,22 @@
 ############################
 FROM golang:1.13-alpine AS builder
 
-# Install git.
+# Install git and gcc
 # Git is required for fetching the dependencies.
-RUN apk update 
-RUN apk add git
-RUN apk add build-base
+# GCC is required for sqlite3
+RUN apk add --no-cache git build-base
```

Comment:
Why no cache?

Reply:
Usage for `--no-cache` is seen [here](https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache).